backend/shared/shared/oidc/mixins.py

```diff
@@ -12,9 +12,9 @@ def __convert_amr_to_list(self, id_token):
         To fix the claim validation issue, convert the id_token amr to list
         when a string is given.
         """
-        if id_token["amr"] and not isinstance(id_token["amr"], list):
+        if id_token.get("amr") and not isinstance(id_token["amr"], list):
```

Comment:
`amr` wasn't there when testing with gdpr-api-tester so added this fail proofing.